### PR TITLE
Adding phpmyadmin

### PR DESCRIPTION
**Pull request by ashleycoles**
Head over to http://localhost:8081/ to see phpmyadmin, on the creds page just leave server empty, and use root and password for username and password.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -21,6 +21,20 @@ services:
...
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    container_name: pma
+    links:
+      - db
+    environment:
+      PMA_HOST: db
+      PMA_PORT: 3306
+      PMA_ARBITRARY: 1
+    restart: always
```
Comment (mporam) [inline docker-compose.yml:33]: can we have restart set to unless-stopped? like the others

File: docker-compose.yml [hunk 2]
```diff
@@ -21,6 +21,20 @@ services:
       MYSQL_ROOT_PASSWORD: password
     restart: unless-stopped
 
+  phpmyadmin:
```
Comment (mporam) [inline docker-compose.yml:24]: does PMA have an error log? if so, can we make sure the column is synced like the other containers?

Comment (ashleycoles) [reply docker-compose.yml:24]: It doesn't! At least not that I could find out about anywhere!

**Review by charliecog — APPROVED** [review]
Looks good!

**Review by mporam — APPROVED** [review]

**Pull request merged**